frontend/src/app/offers/offer-detail/offer-detail.component.html

```diff
@@ -48,11 +48,18 @@ <h3 class="card-header">Zaangażowanie czasowe</h3>
         </div>
         <div class="col-xs-12 col-sm-6 col-md-4">  
             <div class="card text-center">
-                <div class="alert alert-danger">
-                    <h2>Możesz pomóc?</h2><br>
-                    Twoja pomoc jest ważna.<br>
-                    <b><h3>Potrzebujemy Ciebie!</h3></b><br>
-                    <a class="btn btn-light" href="{{ getJoinViewUrl(offer) }}">Zgłoś się na ten wolontariat</a>
+                <div *ngIf="!offer.joined" class="alert alert-danger">
```

Comment:
I believe we cannot use `*ngIf` two times, we should have for example `*ngif="!offer.joined; else joined` here/